master/buildbot/test/integration/test_worker_proxy.py

```diff
@@ -0,0 +1,136 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+import asyncio
+import multiprocessing
+import signal
+import socket
+
+from twisted.internet import defer
+
+from buildbot.test.util.integration import RunMasterBase
+
+from .interop.test_commandmixin import CommandMixinMasterPB
+from .interop.test_compositestepmixin import CompositeStepMixinMasterPb
+from .interop.test_integration_secrets import SecretsConfigPB
+from .interop.test_interruptcommand import InterruptCommandPb
+from .interop.test_setpropertyfromcommand import SetPropertyFromCommandPB
+from .interop.test_transfer import TransferStepsMasterPb
+from .interop.test_worker_reconnect import WorkerReconnect
+
+# This integration test puts HTTP proxy in between the master and worker. You
```

Comment:
well, this comment is obsolete

Reply:
Ah yes. I removed part of this obsolete comment and left the other one... It's fixed now.